neo/Consensus/PrepareRequest.cs

```diff
@@ -8,13 +8,18 @@ namespace Neo.Consensus
 {
     internal class PrepareRequest : ConsensusMessage
     {
+        public uint Timestamp;
```

Comment:
Maybe we can add this `Timestamp` to `ConsensusMessage`  ?

Reply:
Why should we add this to `ConsensusMessage`?